test/unit/Reflection/ReflectionClassTest.php

```diff
@@ -941,26 +954,18 @@ public function testMethodsFromTraits() : void
 
         $classInfo = $reflector->reflect('TraitFixtureC');
 
-        self::assertTrue($classInfo->hasMethod('a'));
+        self::assertFalse($classInfo->hasMethod('a'));
```

Comment:
was this an actual bug? very difficult to see it, given the change in the fixtures: maybe split them instead, so that old tests aren't affected?

Reply:
never mind: original test was indeed wrong here!